lib/smack/SmackRep.cpp

```diff
@@ -13,6 +15,10 @@ const string SmackRep::BOOL_TYPE = "bool";
 const string SmackRep::FLOAT_TYPE = "float";
 const string SmackRep::NULL_VAL = "$NULL";
 
+const string SmackRep::LOAD = "$load.i";
```

Comment:
The `"i"` should not be here, coming instead from concatenation with `int_type(unsigned)` wherever `LOAD` or `STORE` are used.